### ISSUE TITLE
Fix the auth failures in server.log

### ISSUE THREAD

**Issue by ericpromislow** [rancher-sandbox/rancher-desktop#3031]
### Actual Behavior

There are error messages complaining about auth failures in server.log.

### Steps to Reproduce

Run RD and look at server.log

### Result

`Auth failure: user/password validation failure for attempted login of user interactive-user` but there is no apparent breakage.

### Expected Behavior

There shouldn't be auth failures if there's no actual failure.

**Comment by ericpromislow** [comment]
Seeing this in 1.6.1

**Comment by ericpromislow** [comment]
I think this is a symptom of the stale-process problem, where the server from a previous run of RD is still running and handling http requests, but the client is using the new password.

A possible workaround to cope with this is to have the server auth module check the password files when this happens, and reload if it sees a client is using a newer password that matches the one in the file. But it would be better to fix the stale-process problem.

**Comment by jandubois** [comment]
> I think this is a symptom of the stale-process problem, where the server from a previous run of RD is still running and handling http requests, but the client is using the new password.

This is not correct. I have done a "Factory Reset" in 1.6.1 release build and then waited until everything has shut down. After restarting 1.6.1 using all the default options in the first-run dialog I get these messages:

```console
$ cat server.log
2022-10-28T23:55:47.269Z: CLI server is now ready.
2022-10-28T23:55:47.269Z: Credentials server is now ready.
2022-10-28T23:55:55.655Z: Auth failure: user/password validation failure for attempted login of user interactive-user
2022-10-28T23:55:55.704Z: Auth failure: user/password validation failure for attempted login of user interactive-user
```

**Comment by ericpromislow** [comment]
I don't see this commit ```
origin/3031-squelch-bogus-auth-errors

Auth lookup now works with a hash of user=>passwords.
``` from 16:01 Sept 29, 2022.

on the `origin/release-1.6` branch, so this fix didn't get in the 1.6.1 release.

When I run a packaged build off the main branch, the bogus error messages no longer show up.
